src/ebonite/build/builder/base.py

```diff
@@ -32,6 +32,20 @@ def use_local_installation():
         EBONITE_FROM_PIP = tmp
 
 
+@contextmanager
+def use_wheel_installation(path: str):
```

Comment:
add a test for this? just like use this with ._write_distribution and check that file is in the dir (it might be even a txt, not necessary .whl) and that dockerfile contains pip install ...